content/en/docs/external-add-ons/feast/introduction.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Introduction"
+description = "A brief introduction to Feast and Feature Stores"
+weight = 10
++++
+
+## What is Feast?
+
+[Feast](https://docs.feast.dev/) is an [open-source](https://github.com/feast-dev/feast) feature store that helps teams operate ML systems at scale by allowing them to define, manage, validate, and serve features to models in production. 
+
+Feast provides the following functionality:
+
+- __Load streaming and batch data__: Feast is built to be able to ingest data from a variety of bounded or unbounded sources. 
+  Feast allows users to ingest data from streams, object stores, databases, or notebooks. 
+  Data that is ingested into Feast is persisted in both online store and historical stores, which in turn is used for the creation of training datasets and serving features to online systems.
```

Comment:
```suggestion
  Data that is ingested into Feast can be persisted in both online store and historical stores, which in turn is used for the creation of training datasets and serving features to online systems.
```

Reply:
Done in https://github.com/kubeflow/website/pull/3836/commits/a5e72b551cea20a364a90aed4f6ef4f558a2582e